common/src/theme/BlogTagsPostsPage/styles.module.css

```
@@ -0,0 +1,15 @@
+   and the link back to the full tag list. */
+
+.header {
+  padding-top: 4rem;
+}
+
+.title {
+  font-size: 1.5rem;
+  margin-bottom: 0.25rem;
+}
+
+.allTagsLink {
```

Stylelint (17.14.0)

[error] Expected class selector ".allTagsLink" to be kebab-case (selector-class-pattern)

(selector-class-pattern)